Rakefile

```diff
@@ -7,8 +7,7 @@ task :default => :spec
 begin
   require 'github_changelog_generator/task'
   GitHubChangelogGenerator::RakeTask.new :changelog do |config|
-    version = '1.0.0'
-    config.future_release = "v#{version}" if version =~ /^\d+\.\d+.\d+$/
+    config.future_release = '2.0.0'
```

Comment:
Is it worth moving the version number to a single file like we do in some other gems?
eg https://github.com/voxpupuli/hiera-eyaml-gpg/blob/master/lib/hiera/backend/eyaml/encryptors/gpg/version.rb

Reply:
Different implementation, same result. I now load the version from the gemspec which keeps the code simpler.